src/nvim/window.c

```diff
@@ -689,6 +694,24 @@ void win_check_anchored_floats(win_T *win)
   }
 }
 
+/// Return the number of requested fold columns, based on current
+/// folds signs and on user configuration.
+int win_fdccol_count(win_T *wp)
+{
+  const char *fdc = (const char *)wp->w_p_fdc;
+
+  // auto:<NUM>
+  if (strncmp(fdc, "auto:", 5) == 0) {
+    int needed_fdccols;
+    needed_fdccols = getDeepestNesting(wp);
+    int maximum = fdc[5] - '0';
+    return MIN(maximum, needed_fdccols);
```

Comment:
"maximum" is a pretty ceremonious name, why not just "max"? (Only mentioning this for future reference...)

Reply:
done